apps/vs-code-designer/src/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Logic Apps for Visual Studio Code (Standard)
 
-Use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy Logic Apps directly from VS Code.
+In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy logic apps directly to Azure.
```

Comment:
@ccastrotrejo, here's the full text with the new FW link that I created. The link points to the full steps, so please remove the sections "Create your first logic app" and "Run the logic app locally". Thanks!

```suggestion
In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy logic apps directly to Azure. For requirements and more information, see [Create a Standard logic app workflow using Visual Studio Code](https://go.microsoft.com/fwlink/?linkid=2222287).
```